### ISSUE TITLE
additional fields for 'register' 

### ISSUE THREAD

**Issue by shaydahan** [olofk/ipyxact#10]
Hi,

i am using the package for registers description and noticed that you don't include the reset field inside the register type like i have in my ipxact file:
![image](https://user-images.githubusercontent.com/32817885/31586477-38a9175a-b1da-11e7-8794-e936d661d45d.png)

so i edited the ipyxact_yaml.py file as follows:
1. added the "reset" field to the "register" CHILDREN
![image](https://user-images.githubusercontent.com/32817885/31586492-7be10ad2-b1da-11e7-9fa7-573ff4a736fa.png)

2. added a new "reset" field with "value" and "reset" MEMBERS
![image](https://user-images.githubusercontent.com/32817885/31586523-dd6782a4-b1da-11e7-8cf1-35b627c9cd2e.png)

and it is working for me (i am using Python 3.6.3 and ipyxact v0.2.3)

Thanks,
Shay

**Comment by olofk** [comment]
Ah, right. I thought that the reset thing was already added. I looked at it a bit now and it turns out that it was implemented for IP-XACT 2014, where the reset tag is a subtag for each field. Looking at your file, you are using an older IP-XACT version where it was still allowed to set reset for a complete register. I need to think a bit how to handle differences between the IP-XACT versions in a good way.

Thanks for reporting. I will try to come up with a solution that works for all IP-XACT versions
